comp/api/api/apiimpl/grpc.go

```diff
@@ -74,6 +74,10 @@ func (s *serverSecure) TaggerStreamEntities(req *pb.StreamTagsRequest, srv pb.Ag
 	return s.taggerServer.TaggerStreamEntities(req, srv)
 }
 
+func (s *serverSecure) TaggerGenerateContainerIDFromOriginInfo(ctx context.Context, req *pb.GenerateContainerIDFromOriginInfoRequest) (*pb.GenerateContainerIDFromOriginInfoResponse, error) {
```

Comment:
Should we document here or elsewhere that, although this accepts the entire origin information, it currently only processes the external data?

Reply:
I've added a comment in the Remote Tagger but I will also add this here 👍